var/ramble/repos/builtin/applications/lammps/application.py

```diff
@@ -22,7 +23,7 @@ class Lammps(SpackApplication):
     software_spec('impi2018', spack_spec='intel-mpi@2018.4.274')
 
     software_spec('lammps',
-                  spack_spec='lammps@20220623 +opt+manybody+molecule+kspace+rigid',
+                  spack_spec='lammps@20220623.3 +opt+manybody+molecule+kspace+rigid+intel+openmp+openmp-package+asphere+dpd-basic+dpd-meso+dpd-react+dpd-smooth',
```

Comment:
We chatted about this offline, but `+intel` caught my eye. Can you double check this is the right default list of variants?

Reply:
Is it, aside from Intel. I just removed the intel one, since we only need that if we want to use the intel package (which seems like it's used for KNL and other Intel specific optimizations)